pkg/util/customresourcedefinition.go

```diff
@@ -0,0 +1,71 @@
+package util
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	apiextensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
```

Comment:
Why using two different versions together? The API version `apiextensions.k8s.io/v1beta1` is deprecated since v1.16.

Reply:
[karmada is compatible with k8s version 1.15](https://github.com/karmada-io/karmada/blob/master/README.md#kubernetes-compatibility), and there is a little difference between APIs of `apiextensions.k8s.io/v1beta1` and `apiextensions.k8s.io/v1`, the crd created by `apiextensions.k8s.io/v1beta1`  may be wrong now when created with `apiextensions.k8s.io/v1`。